LOG:  00000: statement: update self_ref_trigger set data = 'root!' where id = 1;
LOCATION:  exec_simple_query, postgres.c:938
ERROR:  42P01: relation "self_ref_trigger" does not exist at character 8
LOCATION:  parserOpenTable, parse_relation.c:1160
STATEMENT:  update self_ref_trigger set data = 'root!' where id = 1;
